diesel/Cargo.toml

```diff
@@ -31,6 +31,7 @@ tempdir = "^0.3.4"
 
 [features]
 default = ["with-deprecated"]
+all = ["chrono", "quickcheck", "serde_json", "uuid"]
```

Comment:
Hm. I'd probably call this 'extras'. Is Quickcheck useful for anything but _our_ tests?